src/monitoring_service/blockchain.py

```diff
@@ -100,159 +100,177 @@ def query_blockchain_events(
             from_block=from_block,
             to_block=to_block,
         )
-        for event in monitoring_service_events:
+
+        for event in network_events:
             event_name = event['event']
             block_number = event['blockNumber']
 
-            if event_name == MonitoringServiceEvent.NEW_BALANCE_PROOF_RECEIVED:
+            if event_name == ChannelEvent.OPENED:
                 events.append(
-                    ReceiveMonitoringNewBalanceProofEvent(
-                        token_network_address=event['args']['token_network_address'],
+                    ReceiveChannelOpenedEvent(
+                        token_network_address=event['address'],
                         channel_identifier=event['args']['channel_identifier'],
-                        reward_amount=event['args']['reward_amount'],
+                        participant1=event['args']['participant1'],
+                        participant2=event['args']['participant2'],
+                        settle_timeout=event['args']['settle_timeout'],
+                        block_number=block_number,
+                    )
+                )
+            elif event_name == ChannelEvent.CLOSED:
+                events.append(
+                    ReceiveChannelClosedEvent(
+                        token_network_address=event['address'],
+                        channel_identifier=event['args']['channel_identifier'],
+                        closing_participant=event['args']['closing_participant'],
+                        block_number=block_number,
+                    )
+                )
+            elif event_name == ChannelEvent.BALANCE_PROOF_UPDATED:
+                events.append(
+                    ReceiveNonClosingBalanceProofUpdatedEvent(
+                        token_network_address=event['address'],
+                        channel_identifier=event['args']['channel_identifier'],
+                        closing_participant=event['args']['closing_participant'],
                         nonce=event['args']['nonce'],
-                        ms_address=event['args']['ms_address'],
-                        raiden_node_address=event['args']['raiden_node_address'],
                         block_number=block_number,
                     )
                 )
-            elif event_name == MonitoringServiceEvent.REWARD_CLAIMED:
+            elif event_name == ChannelEvent.SETTLED:
                 events.append(
-                    ReceiveMonitoringRewardClaimedEvent(
-                        ms_address=event['args']['ms_address'],
-                        amount=event['args']['amount'],
-                        reward_identifier=encode_hex(event['args']['reward_identifier']),
+                    ReceiveChannelSettledEvent(
+                        token_network_address=event['address'],
+                        channel_identifier=event['args']['channel_identifier'],
                         block_number=block_number,
                     )
```

Comment:
You could pull `token_network_address`, `channel_identifier` and `block_number` into a `kwargs` dict used for all events.